system/netdata.logrotate.in

```diff
@@ -0,0 +1,15 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		if service netdata status > /dev/null ; then \
+			service netdata restart > /dev/null; \
+		fi;
```

Comment:
restart here gives the tiniest break in the web graphs. Might be slightly annoying I guess, but many things have a very brief break in service at 06:25 when the logs rotate, so it's not unusual.

The best solution would be to signal to the daemon to reopen its log files (-HUP, -USR1, etc). Admit I haven't looked to see if it will currently do this, though.